pyproject.toml

```diff
@@ -29,7 +29,7 @@ dbterd = "dbterd.main:main"
 [tool.poetry.dependencies]
 python = "^3.9"
 click = "^8.1.3"
-dbt-artifacts-parser = "^0.4"
+dbt-artifacts-parser = "^0.5.0"
 
```

Comment:
Hi @Erik-vdg Thanks a lot for the PR.
Do you think it is better to give more free to the top pin: `“^0”`?
See [this](https://python-poetry.org/docs/dependency-specification/) for more details

Reply:
I think that sounds reasonable. Let me make that change.